live-examples/js-examples/bigint/bigint64.js

```diff
@@ -0,0 +1,16 @@
+const buffer = new ArrayBuffer(40);
+const bigint64 = new BigInt64Array(buffer);
+bigint64[0] = BigInt(5886014448488689);
+bigint64[1] = BigInt(1881938909131133);
+bigint64[2] = BigInt(1898875537769492);
+bigint64[3] = BigInt(2156704815705747);
+bigint64[4] = BigInt(3337090060496542);
```

Comment:
```suggestion
bigint64[0] = 5886014448488689n;
bigint64[1] = 1881938909131133n;
bigint64[2] = 1898875537769492n;
bigint64[3] = 2156704815705747n;
bigint64[4] = 3337090060496542n;
```

(Number literals already lose precision)